### ISSUE TITLE
Improve PyTorch Model so that it has at least the MAE of 7 on the validation set

### ISSUE THREAD

**Issue by mdmittriy** [mdmittriy/AgingClock#16]
Figure out how to do this, and write up the things that we should try. Suggestions are welcome!

- [ ]  Play with different hyperparameters (e.g. learning rate, batch size)
- [ ] Try training longer on GPU in Google Cloud Platform

**Comment by mdmittriy** [comment]
From [the paper](https://academic.oup.com/biomedgerontology/article/73/11/1482/4801287) by Zhavoronkov *et al*.
**Grid search** over a space of model's hyperparameters (specifically to find the best performing architecture)
**MAE** Loss function 
**Leaky ReLU** activation after each layer  
**EVE optimizer**  
**Dropout** with 35% after each layer  
Fivefold **cross validation**
